packages/react-components/src/AccessibilityScenarios/Link.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Link } from '@fluentui/react-link';
+
+import { Scenario } from './utils';
+
+export const SiteNavigationAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Site navigation links">
+      <ul>
```

Comment:
Might be good to wrap these in a `<nav aria-label="main menu">` to set up expectations that these are links in a main site navigation bar